R/testthat.R

```diff
@@ -123,13 +186,14 @@ testthat_database <- function(datasource, label = NULL, tests = "default") {
 
   # Create a testing function that lives inside the new testthat env
   run_test <- function(verb, vector_expression) {
-    f <- rlang::parse_expr(vector_expression)
+    f <- parse_expr(vector_expression)
 
     if (verb %in% c("summarise","summarize")) manip <- . %>% summarise(!! f) %>% pull()
     if (verb == "mutate") manip <- . %>% mutate(!! f) %>% pull()
-    if (verb == "arrange") manip <- . %>% arrange(!! f) %>% pull()
+    if (verb == "arrange") manip <- . %>% mutate(new_col = !!f) %>%
+        arrange(!! f) %>% collect() %>% pull("new_col")
     if (verb == "filter") manip <- . %>% filter(!! f) %>% pull()
-    if (verb == "group_by") manip <- . %>% group_by(!! f) %>% summarise() %>% pull()
+    if (verb == "group_by") manip <- . %>% group_by(!! f) %>% summarise() %>% pull() %>% sort()
```

Comment:
Did this finally fixed the out-of-order issue?

Reply:
Sorta.  It did for the `group_by`... still have a bunch of problems with `arrange` and I have no clear direction on how to proceed.  Would love your input (#8).  It might be worth opening a new issue.

Reply:
Is this only impacting MS SQL?

Reply:
Yeah.  Of what I have tested.  I should run it through Oracle.  There are problems getting the tests to run on a few other databases.